src/app/account/account-reports/account-report-setup/better-plants-setup/better-plants-setup.component.ts

```diff
@@ -29,6 +29,7 @@ export class BetterPlantsSetupComponent {
   isFormChange: boolean = false;
   itemToEdit: IdbAccountAnalysisItem;
   selectedAnalysisItem: IdbAccountAnalysisItem;
+  baselineYearWarning: string;
   constructor(private accountReportDbService: AccountReportDbService,
     private accountReportsService: AccountReportsService,
     private dbChangesService: DbChangesService,
```

Comment:
Looks good. Just need to set the report baseline year based on the analysis item that is selected. Within the save function add this line (probably line 62):

`selectedReport.baselineYear = this.selectedAnalysisItem.baselineYear;`